src/controllers/userController.js

```diff
@@ -0,0 +1,90 @@
+import { save, getById, update, getAutoSuggestUsers, getAllUsers, remove } from '../services/UserService';
+import { v4 as uuidv4 } from 'uuid';
+import { bad } from '../util/constant';
+import logger from '../util/logger';
+
+export const saveUser = async (request, response) => {
+    const userId = uuidv4();
+    const userLogin = request.body.login;
+    const userPassword = request.body.password;
+    const userAge = request.body.age;
+    const isUserDeleted = false;
+    logger.info(`saveUser {id: ${userId}, login: ${userLogin}, password: ${userPassword}, age: ${userAge}, isDeleted: ${isUserDeleted}}`);
+    try {
+        const user = await save(
+            { id: userId, login: userLogin, password: userPassword, age: userAge, isDeleted: isUserDeleted });
+        response.status(201).send(user);
+    } catch (err) {
+        logger.error(`saveUser failed: {id: ${userId}, login: ${userLogin}, password: ${userPassword}, age: ${userAge}, isDeleted: ${isUserDeleted}}, message: ${err.message}`);
+        response.status(500).send(bad);
+    }
+};
+
+export const getUser = async (request, response) => {
+    const id = request.params.id;
+    logger.info(`getUser by id: ${id}`);
+    try {
+        const user = await getById(id);
+        if (user) {
+            response.send(user);
+        } else {
+            response.status(404).send();
```

Comment:
Empty response body is not a best solution. It is preferable to add error message or json response body to make it clear what caused this error

Reply:
OK. I done it.